qdrant-landing/content/documentation/integrations/aleph-alpha-search.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Multimodal Search with Aleph Alpha
+weight: 1100
+---
+
+# Multimodal Semantic Search with Aleph Alpha 
+
+Semantic search goes well beyond textual data. It is surely a great alternative for the full-text search 
+but also allows us to perform a reverse image search and many more. However, in most cases, we were limited 
+to using the same data type for both documents and queries (text-text, image-image, audio-audio, etc.). With 
+the recent growth of multimodal architectures, it became possible to encode different data types into the same 
+latent space — for example, texts and images. That opens up some great possibilities, as we can finally **explore 
+non-textual data, for example visual, with text queries**. In the past, it would require labelling every image 
+with a description of what it presents. Right now, we can rely on vector embeddings, which can represent all 
+the inputs in the same space.
+
+![](/docs/integrations/aleph-alpha/2d_text_image_embeddings.png)
+
+*Two examples of text-image pairs presenting a similar object, encoded by a multimodal network into the same 
+2D latent space. Both texts are examples of English [pangrams](https://en.wikipedia.org/wiki/Pangram). 
+https://deepai.org generated the images with pangrams used as input prompts.*
+
+## Sample dataset
+
+You will be using [COCO](https://cocodataset.org/), a large-scale object detection, segmentation, and captioning dataset. It provides 
+various splits, 330K images in total. For the demonstration purposes we can choose a 
+[2017 validation split](http://images.cocodataset.org/zips/train2017.zip) that contains 5K images from different 
+categories.
+
+## Prerequisites
+
+There is no need to curate your datasets and train the models. [Aleph Alpha](https://www.aleph-alpha.com/), already has multimodality and multilinguality already built-in. There is an [official Python client](https://github.com/Aleph-Alpha/aleph-alpha-client) that simplifies the integration. [can be integrated seamlessly](https://qdrant.tech/documentation/integrations/#aleph-alpha).
+
+In order to enable the search capabilities, you need to build the search index to query on. For our purposes, 
+we are going to vectorize the images and store their embeddings along with the filenames, we can then return the most 
+similar files for given query. There are two things we need to set up before we start though:
+
+1. A Qdrant instance has to be running. If you want to launch it locally,
+   [Docker is the fastest way to do that](https://qdrant.tech/documentation/quick_start/#installation).
+2. You need to have a [Aleph Alpha account registered and confirmed](https://app.aleph-alpha.com/). Once it’s done, 
+   the API key has to be created (see: [API Tokens](https://app.aleph-alpha.com/profile)).
```

Comment:
Perhaps use active voice, i.e. "Once that's done, create an API key (...)" here.